Polymorhism/PleinTextForrmater.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.Oom.ObjectModel
+{
+    internal class PleinTextForrmater : PleinTextForrmater
```

Comment:
Niste definirali interface `IPlainTextFormatter` kojeg trebaju klase TextFormatter i HtmlFormatter implementirati (naslijediti). On bi trebao izgledati ovako:
```
namespace Vsite.Oom.ObjectModel
{
    internal interface ITextFormatter
    {
        public string FormatParagraph(string text);
        public string FormatHeading1(string text);
        public string FormatHeading2(string text);
    }
}
```
Ova klasa i `HtmlTextFormatter` moraju biti izvedene iz tog interfacea. Pogledajte kako je to riješila Vaša kolegica.